devel/dotnet-sdk/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dotnet-sdk
+version             5.0.202
+revision            0
+categories          devel dotnet
+license             MIT
+maintainers         {tsabirgaliev @tsabirgaliev} openmaintainer
+
+description         Core functionality needed to create .NET Core projects, that is shared between Visual Studio and CLI
+long_description    .NET is a free, cross-platform, open source developer platform \
+                    for building many different types of applications. \
+                    \
+                    With .NET, you can use multiple languages, editors, and libraries \
+                    to build for web, mobile, desktop, games, and IoT.
+
+homepage            https://dotnet.microsoft.com/
+platforms           darwin
+distname            ${name}-${version}-osx-x64
+master_sites        https://download.visualstudio.microsoft.com/download/pr/a7d92db4-5f1f-4478-9caf-6e45c6ac8b09/2363d553132aaed687265c563431673b/
```

Comment:
I advise moving the hash from the link into a separate variable and place it after the version with a comment. I think it will be so more convenient for you to update the port in the future.

```tcl
version 5.0.202
set ver_hash a7d92db4-5f1f-4478-9caf-6e45c6ac8b09/2363d553132aaed687265c563431673b
# Can be found in https://dotnet.microsoft.com/download/dotnet/5.0

...

master_sites https://download.visualstudio.microsoft.com/download/pr/${ver_hash}/
```

Reply:
@judaew yep, good point